exercises/practice/largest-series-product/largest_series_product_test.cpp

```diff
@@ -0,0 +1,67 @@
+ #include "largest_series_product.h"
+ #ifdef EXERCISM_TEST_SUITE
+ #include <catch2/catch.hpp>
+ #else
+ #include "test/catch.hpp"
+ #endif
```

Comment:
These preprocessor instructions in lines 1-6 have a leading space. Is that on purpose?